autoload/phpactor.vim

```diff
@@ -435,14 +435,41 @@ endfunction
 " RPC -->-->-->-->-->--
 """""""""""""""""""""""
 
+let g:phpactorProjectRootPatterns = ['composer.json', '.git']
```

Comment:
- Let's add these in https://github.com/phpactor/phpactor/blob/d5265a951494fe267e98b0a9956d83b35fbad06c/ftplugin/php.vim#L41 with some documentation (so it will find it's way into the VIM help.
- Add `.phpactor.json` and `.phpactor.yml` here?

Reply:
Done in ftplugin, I removed this redundant part.